axiom/userbase.py

```diff
@@ -268,6 +275,33 @@ def addLoginMethod(self, localpart, domain, protocol=ANY_PROTOCOL, verified=Fals
                                internal=internal)
 
 
+    def setPassword(self, newPassword):
+        """
+        Set this account's password unconditionally.
+
+        @param newPassword: The new password.
+
+        @return: A deferred firing when the password has been changed.
+        """
+        self.password = unicode(newPassword)
```

Comment:
I think we need to handle `None` specially here. See `LoginBase.addAccount`.

Reply:
Done.